### ISSUE TITLE
Configure on facets appending values

### ISSUE THREAD

**Issue by proudlygeek** [algolia/react-instantsearch#2077]
**Describe the bug 🐛**

Declaring `<Configure facets="*" />` seems to not be taken into consideration at query time; more specifically, the value sent at query time is `*facet1,facet2...facetN` rather than just `*`.

**To Reproduce 🔍**

1. Add a `<Configure facets="*" />` tag
2. Send any query
3. Check the network tab for the actual query (see screenshot)

Code Sandbox Example:

[![Edit react-instantsearch-app](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/z21pp7y3vp?codemirror=1&fontsize=14)

**Expected behavior 💭**

`<Configure facets="*" />` should have priority over any other facets

**Screenshots 🖥**

![image](https://user-images.githubusercontent.com/357937/53183811-4d836c00-35fc-11e9-8466-8ed48d21902b.png)

**Environment:**

 - OS: Mac
 - Browser: Chrome
 - Version: 72

**Comment by Haroenv** [comment]
There's two possible solutions for this:

```js
            <Configure facets={['*']} />
```

```js
            <Configure facets="*," />
```

https://codesandbox.io/s/omo3qj8q5

**Comment by samouss** [comment]
It’s probably more related to the helper no?

**Comment by Haroenv** [comment]
You're right, InstantSearch.js has the issue too, so I'll move it https://codesandbox.io/s/0okx84q4z0
